modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -136,6 +136,31 @@ def test_read_multiple_csv_s3():
     df_equals(modin_df, pandas_df)
 
 
+@pytest.mark.skipif(
+    Engine.get() != "Ray",
```

Comment:
Shouldn't we also check that we're in experimental mode?
```suggestion
    get_current_execution() != "ExperimentalPandasOnRay",
```

Reply:
This test is already placed in `modin/experimental` and experimental mode is implied. Changes makes sense, but the condition should be modified throughout all `test_io_exp.py` to be uniform.